thirdparty/clamav-db/zimbra-clamav-db/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: zimbra-clamav-db
+Section: mail
+Priority: optional
+Maintainer: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.9.5
+
+Package: zimbra-clamav-db
+Architecture: all
+Depends: zimbra-base
+Description: Inital ClamAV Databases for ClamAV
```

Comment:
I'm late to the party, and I see Gren noticed this but it didn't get fixed so I'll comment as well here...
- typo: change 'Inital' to 'Initial' here and on line below.